server/routes/playlist.js

```
@@ -101,4 +113,30 @@ router.get('/get-track-vector', jwt.verifyJWT, async (req, res, next) => {
   }
 })
 
+/* adds playlist as a favorite playlist */
+router.post('/add-favorite', jwt.verifyJWT, async (req, res, next) => {
+  try {
+    const userId = req.userId
+    const { playlistId } = req.body
+
+    await Playlist.findOneAndUpdate({ userId, playlistId }, { favorite: true })
+    res.status(200).json()
+  } catch (error) {
+
```

This should probably be a 500 response. Empty catches are a _code smell_.